docs/02.API-REFERENCE.md

```diff
@@ -696,29 +696,51 @@ typedef jerry_value_t (*jerry_external_handler_t) (const jerry_call_info_t *call
 
 - [jerry_create_external_function](#jerry_create_external_function)
 
+## jerry_value_free_callback_t
+
+**Summary**
+
+Native free callback of generic value types.
+
+*Note*:
+ - Referred values by this method must have at least 1 reference. (Correct API usage satisfies this condition)
+
+**Prototype**
+
+```c
+typedef void (*jerry_value_free_callback_t) (void *native_p);
```

Comment:
Was wondering if it would make sense to have a "type" information as second argument for this callback. What do you think? Could this be a useful feature? This "type" could/would indicate what kind of data should be freed.

Reply:
I am neutral. If others think this is a good idea we could do it in another patch (before release).

Reply:
That's true. We could even do a different WIP PR to see how it would look.